cmd/ignite/cmd/cmdutil/flags.go

```diff
@@ -20,3 +21,8 @@ func AddInteractiveFlag(fs *pflag.FlagSet, interactive *bool) {
 func AddForceFlag(fs *pflag.FlagSet, force *bool) {
 	fs.BoolVarP(force, "force", "f", *force, "Force this operation. Warning, use of this mode may have unintended consequences.")
 }
+
+func AddSSHFlags(fs *pflag.FlagSet, identityFile *string, timeout *uint32) {
+	fs.StringVarP(identityFile, "identity", "i", "", "Override the vm's default identity file")
+	fs.Uint32Var(timeout, "timeout", constants.SSH_DEFAULT_TIMEOUT_SECONDS, "Timeout waiting for connection in seconds")
```

Comment:
Should this be `--ssh-timeout`?
Since this is a VM creation flag, there could be other timeouts